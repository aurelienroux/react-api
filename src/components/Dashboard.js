import React from "react";
import { Link } from "react-router-dom";

const Dashboard = () => {
  return (
    <div>
      <Link to="/sites">Sites</Link>
      <Link to="/clients">Clients</Link>
    </div>
  );
};

export default Dashboard;
