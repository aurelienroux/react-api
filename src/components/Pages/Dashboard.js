import React from "react";
import { Link } from "react-router-dom";
import { makeStyles } from "@material-ui/core/styles";
import { Container, Paper, Typography } from "@material-ui/core";

const useStyles = makeStyles(theme => ({
  container: {
    display: "flex",
    justifyContent: "space-around",
    padding: 50
  },
  paper: {
    padding: theme.spacing(10, 20)
  }
}));

const Dashboard = () => {
  const classes = useStyles();

  return (
    <Container className={classes.container}>
      <Link to="/sites">
        <Paper className={classes.paper}>
          <Typography variant="h4">Sites</Typography>
        </Paper>
      </Link>
      <Link to="/clients">
        <Paper className={classes.paper}>
          <Typography variant="h4">Clients</Typography>
        </Paper>
      </Link>
    </Container>
  );
};

export default Dashboard;
