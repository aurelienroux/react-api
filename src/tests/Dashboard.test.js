import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { create } from "react-test-renderer";
import Dashboard from "../components/Pages/Dashboard";

describe("Button component", () => {
  test("Matches the snapshot", () => {
    const dashboard = create(
      <Router>
        <Dashboard />
      </Router>
    );
    expect(dashboard.toJSON()).toMatchSnapshot();
  });
});
