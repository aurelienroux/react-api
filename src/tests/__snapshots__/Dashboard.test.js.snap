// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<div
  className="MuiContainer-root makeStyles-container-1 MuiContainer-maxWidthMd"
>
  <div
    className="dashboard"
  >
    <a
      className="dashboard__card"
      href="/sites"
      onClick={[Function]}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 makeStyles-paper-2 MuiPaper-rounded"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Sites
        </h4>
      </div>
    </a>
    <a
      className="dashboard__card"
      href="/clients"
      onClick={[Function]}
    >
      <div
        className="MuiPaper-root MuiPaper-elevation1 makeStyles-paper-2 MuiPaper-rounded"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Clients
        </h4>
      </div>
    </a>
  </div>
</div>
`;
